docs/howto/import.md

```diff
@@ -16,47 +16,27 @@ redirect_from:
 
 ## Zero-copy import
 
```

Comment:
I think this page should open with a quick explanation of the two options (zero-copy / copy), what they are, and what the differences are / what would drive the decision to use one instead of the other


You could then recap this in the doc here as to what zero-copy import is.

Reply:
Done